src/remembrall/classes/Fields.py

```diff
@@ -72,3 +72,19 @@ def __init__(self, value: str):
 
     def __str__(self):
         return datetime.strftime(self.value, format)
+    
+class Address(Field):
+    def __init__(self, value: str, existing_address=None):
+        if existing_address is not None:
```

Comment:
мені здається, ти дуже накрутив. Задум зрозумів, і тепер зрозумілі інстанси, але можна буже просто. Можеш залишити як є, бо тоді треба міняти і в інших місцях
```suggestion
    def __init__(self, value: str):
        if self.value:
```